netbout-web/src/main/java/com/netbout/email/EmMessages.java

```diff
@@ -159,4 +160,11 @@ private void email(final Friend friend, final String text)
         );
     }
 
+    /**
+     * Generates a random integer.
+     * @return Random integer.
+     */
+    private int generateRandomId() {
+        return (int) (Math.random() * Integer.MAX_VALUE);
```

Comment:
@antonini do we really need random here? Maybe, timestamp would be enough?
```java
System.nanoTime()
```
What do you think?